test/functional-portable/datastoresrp/noncloud/resources/mongodb_test.go

```diff
@@ -70,49 +70,3 @@ func Test_MongoDB_Manual(t *testing.T) {
 
 	test.Test(t)
 }
-
-// Test_MongoDB_Recipe validates:
-// the creation of a mongoDB from a recipe that uses an Azure resource
-func Test_MongoDB_Recipe(t *testing.T) {
```

Comment:
Moved this test to the **cloud** package.

Reply:
Thank you! 🙇